apollo-router-core/src/query_planner/mod.rs

```diff
@@ -218,7 +221,11 @@ impl PlanNode {
                         )
                         .instrument(tracing::info_span!(
                             "fetch",
-                            "otel.kind" = %SpanKind::Internal
+                            "otel.kind" = %SpanKind::Client,
+                            "net.peer.name" = &field::Empty,
```

Comment:
do we need to add the net and http info if it's empty?

Reply:
It's not empty, it's updated inside the span. But we always need to declare all the record we could have in a span, we can't add them dynamically, only edit them dynamically